docs/build.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+# Copy in service RAML files 
+
+cp ../core/data/raml/*.raml ./core/data/
+cp ../core/metadata/raml/*.raml ./core/metadata/
+cp ../core/command/raml/*.raml ./core/command/
+cp ../support/logging/raml/*.raml ./support/logging/
+cp ../export/client/raml/*.raml ./export/client/
+
+# Build image (copying in documentation sources)
+
+docker build -t doc-builder:latest -f Dockerfile.build .
+if [ -d _build ]
```

Comment:
No need to test if _build exists, just run the `rm -rf`

Reply:
OK, Will do.